File: archetype-base/src/test/java/com/alo7/archetype/log/LogMessageBuilderTest.java
// Copyright 2019 Alo7 Inc. All rights reserved.

package com.alo7.archetype.log;

import org.junit.Assert;
import org.junit.Test;

/**
 * @author Kelin Tan
 */
public class LogMessageBuilderTest {
    @Test
    public void testLogBuilder() {
        Assert.assertEquals(LogMessageBuilder.builder().message("this a test")
                .parameter("key1", "value1")
                .parameter("key2", "key2")
                .build(), "this a test Parameters: key1=value1,key2=key2");
    }
}

File: archetype-common/src/test/java/com/kelin/archetype/common/log/LogMessageBuilderTest.kt
// Copyright 2020 Kelin Inc. All rights reserved.

package com.kelin.archetype.common.log

import com.kelin.archetype.test.KtTestUtils
import org.junit.jupiter.api.Test

/**
 * @author Kelin Tan
 */
class LogMessageBuilderTest : KtTestUtils {
    @Test
    fun testLogBuilder() {
        LogMessageBuilder.builder().message("this a test")
            .parameter("key1", "value1")
            .parameter("key2", "key2")
            .build() eq "this a test Parameters: key1=value1,key2=key2"
    }
}